changelogs/DP-17154.yml

```diff
@@ -0,0 +1,28 @@
+Changed:
```

Comment:
Does this changelog mean that DP-17154 changes are all in this PR?

Reply:
This is listed as a changed file because this PR's base branch is `patternlab/DP-17200-add-horizontal-nav-code-branch` (= the integration branch).  And, this branch is created from DP-17154's whose base branch is also `patternlab/DP-17200-add-horizontal-nav-code-branch`.